packages/hierarchy-browser/src/common/dataProviders/CommunityDataProvider.ts

```diff
@@ -136,17 +169,13 @@ export class CommunityDataProvider {
 			offset: offset,
 			filtered: this._filterEntitiesFlag,
 		}
-		if (this._loadNeighborsCallback) {
-			try {
-				const nextNeghbors = this.loadNeighborsAsync(params)
-				return nextNeghbors || []
-			} catch (err) {
-				throw Error(
-					`Error: There is an issue loading neighbors for level ${this._level}, community ${this._community}`,
-				)
-			}
-		} else {
-			return this.loadStaticNeighborCommunities(params)
+		try {
+			const nextNeghbors = this.loadNeighborsAsync(params)
+			return nextNeghbors || []
+		} catch (err) {
+			throw Error(
```

Comment:
this swallows the source error, which may be useful for debugging - maybe we should log it out before throwing a new Error?